src/control/server/storage/bdev/backend_test.go

```diff
@@ -416,3 +421,75 @@ func TestBdevBackendUpdate(t *testing.T) {
 		})
 	}
 }
+
+func TestBdev_Backend_cleanHugePages(t *testing.T) {
```

Comment:
Would be interesting to test the case where there is a subdirectory with the selected prefix.